text/0047-reset-stop-words-synonyms-settings-with-null.md

```diff
@@ -0,0 +1,68 @@
+- Title: Reset Stop-words and Synonyms settings with null value
+- Start Date: 2021-06-08
+- Specification PR: [#47](https://github.com/meilisearch/specifications/pull/47)
+- MeiliSearch Tracking-issues:
+
+# Reset Stop-words and Synonyms settings with a null value.
+
+## 1. Functional Specification
+
+### I. Summary
+
+All Sub Settings `POST` endpoint accepts a null value in the http request body to reset the engine configuration to the default values. This is not the case for [Stop-words](https://docs.meilisearch.com/reference/api/stop_words.html#update-stop-words) and [Synonyms](https://docs.meilisearch.com/reference/api/synonyms.html#update-synonyms) settings endpoints.
```

Comment:
whats happens if you send null today?

Reply:
It resets the settings. Except for stop-words and synonyms

Reply:
If you send null today on the route you get an error about the request body format